package.json

```diff
@@ -17,12 +22,12 @@
   },
   "homepage": "https://github.com/npm/are-we-there-yet",
   "devDependencies": {
-    "standard": "^11.0.1",
-    "tap": "^12.0.1"
+    "@npmcli/lint": "^1.0.2",
+    "tap": "^15.0.9"
   },
   "dependencies": {
     "delegates": "^1.0.0",
-    "readable-stream": "^2.0.6"
+    "readable-stream": "^3.6.0"
```

Comment:
This semver major had to do w/ changes in node versions 9 and 10. Since 10 is already the minimum supported version for this repo this will only need to constitute a patch release.